### ISSUE TITLE
Calibrator is missing tile size

### ISSUE THREAD

**Issue by bennahugo** [SpheMakh/Stimela#106]
@SpheMakh Running into problems with meerkat data:
```
1066.97 274.7Gb gainopts(StefCal.py:1287:run_gain_solution): iter 1: 82.03% (33034/40273) conv, 0 gfs, max update 0.994646
```
Need to figure out how to pass tile sizes to stefcal, because this is only a 7 hour observation with 11 antennas

**Comment by bennahugo** [comment]
Okay option is **ms_sel.tile_size** type int. I will add this and push a fix tomorrow

**Comment by SpheMakh** [comment]
Fixed in #107
